qdrant-landing/content/documentation/data-ingestion-beginners.md

```diff
@@ -298,21 +237,22 @@ if __name__ == "__main__":
         text_embedding, image_embedding, points, text_review, product_image = [], [], [], "", ""
         for idx, doc in enumerate(docs):
             source = doc.metadata['source']
-            if source.endswith(".txt"):
-                text_review, text_embedding = process_document(doc)
+            if source.endswith(".txt") or source.endswith(".pdf"):
+                _text_review_source, text_review = process_text(doc)
             elif source.endswith(".png"):
-                product_image, image_embedding = process_document(doc)
+                product_image_source, product_image = process_image(doc)
         if text_review:
-            point = PointStruct(
+            point = models.PointStruct(
                 id=idx,  # Unique identifier for each point
                 vector={
-                    "text_embedding": text_embedding[0],                      
-                    "image_embedding": image_embedding[0].tolist(), 
+                    "text_embedding": models.Document(
+                        text=text_review, model="sentence-transformers/all-MiniLM-L6-v2"
+                    ),
+                    "image_embedding": models.Image(
+                        image=product_image, model="Qdrant/clip-ViT-B-32-vision"
+                    ),
```

Comment:
We've replaced embeddings with `models.Document` and `models.Image`, however, the explanation of the code below doesn't have any details about these structures, but only about embeddings, maybe we could write a couple of lines that embedding computation happens implicitly with fastembed integration